File: Praktikum1 C/PRAK008-2510817220002-AishaZwarintyanda.c
#include <stdio.h>

int main() {
    int jarak = 14;
    int putaran = 5;
    float pi = 3.14;
    float jj;

    printf("Diketahui:\n");
    printf("Pak Dengklek mengelilingi taman = %d Putaran\n", putaran);
    printf("Jarak tempuh Pak Dengklek = %d Kilometer\n", jarak);
    printf("Jawaban:\n");

    //rumus jari-jari
    jj = jarak / (putaran * 2 * pi);

    printf("Jari-jari taman yang dikelilingi Pak Dengklek adalah %.2f Kilometer\n", jj);

    return 0;
}

File: Praktikum1 Python/PRAK008-2510817220002-AishaZwarintyanda.py
jarak = 14
putaran = 5
pi = 3.14

print("Diketahui:")
print(f"Pak Dengklek mengelilingi taman = {putaran} Putaran")
print(f"Jarak tempuh Pak Dengklek = {jarak} Kilometer")
print("Jawaban:")

#Rumus jari-jari
jj = jarak / (putaran * 2 * pi)

print(f"Jari-jari taman yang dikelilingi Pak Dengklek adalah {jj:.2f} Kilometer")